docs/release/trg-0/trg-8-03.md

```diff
@@ -0,0 +1,257 @@
+---
+title: TRG 8.03 - KIT Artefacts
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+This explains each artefacts a KIT can or must follow - depending on the graduation stage (TRG 8.02)
+
+## CHANGELOG
+
+To achieve some uniformity, we prescribe a versioning scheme for KITs. Here we use a modified semantic versioning. With semantic versioning, the version is composed of three parts as follows:
+
+`MAJOR.MINOR.PATCH`
```

Comment:
To be honest, this does not at all make sense to me. 
Either we follow semantic versioning in it's full extend, or we invent a different scheme, that works for the phasing concept. 
Being that close to semantic versioning, but bumping versions on phases, rather semantics will only confuse people. They will see the schematic and directly associate it with semantic versioning